devicetypes/smartthings/zwave-switch.src/zwave-switch.groovy

```diff
@@ -187,7 +188,7 @@ def poll() {
   * PING is used by Device-Watch in attempt to reach the Device
 **/
 def ping() {
-		refresh()
```

Comment:
@mckeed do you think we need `zwave.manufacturerSpecificV1.manufacturerSpecificGet().format()` in `poll` or could that be removed as well?

Reply:
Poll isn't actually called on this device type. The Polling capability should be removed. https://smartthings.atlassian.net/browse/DVCSMP-1524